packages/site/src/utils/snap.ts

```diff
@@ -104,4 +128,41 @@ export const getListCredentialRequest = async () => {
   });
 };
 
+export const signMessage = async (authRequestToSignBase64: string) => {
+  try {
+    const permissions: any[] = await window.ethereum.request({
+      method: 'wallet_requestPermissions',
+      params: [{ eth_accounts: {} }],
+    });
+    console.log('permissions', permissions);
+
+    const accountsPermission = permissions.find(
+      (permission) => permission.parentCapability === 'eth_accounts',
```

Comment:
I suspect in the latest snap eth_accounts is removed.  Please, double check that is's working